Library/Homebrew/formulary.rb

```diff
@@ -57,6 +57,7 @@ def self.class_s(name)
     class_name = name.capitalize
     class_name.gsub!(/[-_.\s]([a-zA-Z0-9])/) { $1.upcase }
     class_name.tr!("+", "x")
+    class_name.gsub!(/\b[@]\b/, "AT")
```

Comment:
`/.@\d` matches `l@1` in this case, which produces things like:
```
Expected to find class OpenssAT1, but only found: OpensslAT11.
```
I'm not sure how to shake that off in the context of `gsub`.